README.md

```diff
@@ -160,6 +262,114 @@ of how some key tasks can be carried out using Valet:
 - [NPM](https://docs.npmjs.com/cli) commands may be executed by using `npm <command>`.
 - [Artisan](https://laravel.com/docs/10.x/artisan) commands may be executed by using `php artisan <command>`.
 
+
+### Local development setup using docker compose:
+1. Install docker according to your platform instructions found [here](https://docs.docker.com/get-docker/).
+2. Clone the repository:
+
+    ```bash
+    git clone https://github.com/accessibility-exchange/platform.git
+    cd platform
+    ```
+
+3. Create a `.env` file from the included example file:
+
+    ```bash
+    cp .env.local.example .env
+    ```
+   
+    Then, change the `APP_ENV` value to `local`:
+
+    ```dotenv
+    APP_ENV=local
+    ```
+
+4. Generate an encryption key for [CipherSweet](https://github.com/spatie/laravel-ciphersweet):
+
+    ```bash
+    docker run --rm -it alpine apk add openssl && openssl rand -hex 32
+    ```
+
+    Add it to your `.env` file:
+    
+    ```dotenv
+    CIPHERSWEET_KEY="<your key>"
+
+5. Build you application container that will be used to generate keys:  
+
+    ```bash
+    docker compose -f docker-compose.local.yml build platform.test
+    ```
+    ```
+
+6.  Generate an application key:
+
+    ```bash
+    docker run --rm --entrypoint '' platform.test php artisan key:generate --show
+    ```
```

Comment:
I'm not sure why but this image is actually called `platform-platform.test` for me. So the above command other's that refer to the image name all fail. I'm not sure if this is a problem with the documentation here or something specific to my system, but I did prune out everything in docker and it still came up as I mentioned.

Reply:
I have fixed the command. I deleted all local images and volumes and stepped through each step to recreate and fixed all the steps.